lib/msf/core/exploit/remote/ldap.rb

```diff
@@ -8,18 +8,23 @@
 
 module Msf
   module Exploit::Remote::LDAP
+    include Msf::Auxiliary::Report
```

Comment:
I updated the authenticator to use include this now https://github.com/rapid7/metasploit-framework/blob/ab55abd7714091d4d1712f9c33cc22d405e2b70c/lib/msf/core/exploit/remote/kerberos/service_authenticator/base.rb#L9

I think that's what it was added here at least 😄